### ISSUE TITLE
Feature request: Basic editor instead of "add answer" / answer buttons

### ISSUE THREAD

**Issue by onword** [learnweb/moodle-mod_moodleoverflow#83]
Hello, first of all, wonderful plugin, thank you!

As the upvote system mirrors reddit and similar sites, I would like to request that instead of seeing an "answer" link or "add an answer" button, that when a user clicks into a discussion topic they are immediately presented with an inline editor.

![image](https://user-images.githubusercontent.com/78563574/122082404-f9ed4b80-cdcd-11eb-9cc7-bb177b3ce7fa.png)

**Comment by pejhab** [comment]
I second that. It's now the norm on the Internet and expected by all users.

**Comment by luethen** [comment]
I agree, it would be nice to see the question while writing an answer! At the moment, the editor opens in a new page and the question is not visible anymore.

**Comment by NinaHerrmann** [comment]
Feel free to submit a pull request :) especially @pejhab as it is expected.

**Comment by pejhab** [comment]
@NinaHerrmann, I certainly would it it were within my technical know-how. As it is, I am afraid someone else will have to do it. :(
